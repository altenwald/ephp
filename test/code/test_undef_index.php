<?php

error_reporting(E_ALL);

$wp_filter = [];
print $wp_filter[100]["hey"];

print "OK\n";
