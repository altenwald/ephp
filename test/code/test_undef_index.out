
Notice: Undefined offset 100 in {{CWD}}/test/code/test_undef_index.php on line 6
OK
